config/bidderinfo_test.go

```diff
@@ -377,8 +371,66 @@ func TestProcessAliasBidderInfo(t *testing.T) {
 			Tracker:  "alias-tracker",
 		},
 	}
-	bidderB := parentBidderInfo
+	bidderB := parentWithSyncerKey
 	bidderB.AliasOf = "bidderA"
+	bidderB.Syncer = &Syncer{
+		Key: bidderB.Syncer.Key,
+	}
+
+	parentWithoutSyncerKey := BidderInfo{
+		AppSecret: "app-secret",
+		Capabilities: &CapabilitiesInfo{
+			App: &PlatformInfo{
+				MediaTypes: []openrtb_ext.BidType{openrtb_ext.BidTypeBanner, openrtb_ext.BidTypeVideo, openrtb_ext.BidTypeNative},
+			},
+			Site: &PlatformInfo{
+				MediaTypes: []openrtb_ext.BidType{openrtb_ext.BidTypeBanner, openrtb_ext.BidTypeVideo, openrtb_ext.BidTypeNative},
+			},
+		},
+		Debug: &DebugInfo{
+			Allow: true,
+		},
+		Disabled:            false,
+		Endpoint:            "https://endpoint.com",
+		EndpointCompression: "GZIP",
+		Experiment: BidderInfoExperiment{
+			AdsCert: BidderAdsCert{
+				Enabled: true,
+			},
+		},
+		ExtraAdapterInfo: "extra-info",
+		GVLVendorID:      42,
+		Maintainer: &MaintainerInfo{
+			Email: "some-email@domain.com",
+		},
+		ModifyingVastXmlAllowed: true,
+		OpenRTB: &OpenRTBInfo{
+			GPPSupported: true,
+			Version:      "2.6",
+		},
+		PlatformID: "123",
+		Syncer: &Syncer{
+			IFrame: &SyncerEndpoint{
+				URL:         "https://foo.com/sync?mode=iframe&r={{.RedirectURL}}",
+				RedirectURL: "https://redirect/setuid/iframe",
+				ExternalURL: "https://iframe.host",
+				UserMacro:   "UID",
+			},
+		},
+		UserSyncURL: "user-url",
+		XAPI: AdapterXAPI{
+			Username: "uname",
+			Password: "pwd",
+			Tracker:  "tracker",
+		},
+	}
```

Comment:
This is correct. Suggestion - might wanna check if you could reduce `BidderInfo` struct fields to only include syncer since other fields seems un-important for this test case.

Reply:
Yeah you can remove the other fields, just tested it myself.

Reply:
https://github.com/prebid/prebid-server/pull/3082/commits/8b64cd6cc1218afa2b21c55d4916eeaeb451f9ab makes suggested changes